File: ejercicio_java/src/main/java/com/utpl/tarea/Tarea.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Project/Maven2/JavaApp/src/main/java/${packagePath}/${mainClassName}.java to edit this template
 */

package com.utpl.tarea;

/**
 *
 * @author Skous
 */
public class Tarea {
class Universidad {
    String nombre;
    String ubicacion;

    public Universidad(String nombre, String ubicacion) {
        this.nombre = nombre;
        this.ubicacion = ubicacion;
    }
}

class Carrera {
    String nombre;
    Universidad universidad;

    public Carrera(String nombre, Universidad universidad) {
        this.nombre = nombre;
        this.universidad = universidad;
    }
}

// Ejemplo de uso
Universidad miUniversidad = new Universidad("UTPL", "Loja");
Carrera miCarrera = new Carrera("Transformacion Digital De Empresas", miUniversidad);

    public static void main(String[] args) {
    }
}

File: ejercicio_python/Tarea.py
class Universidad:
    def __init__(self, nombre, ubicacion):
        self.nombre = nombre
        self.ubicacion = ubicacion

class Carrera:
    def __init__(self, nombre, universidad):
        self.nombre = nombre
        self.universidad = universidad

# Ejemplo de uso
mi_universidad = Universidad("UTPL", "Ciudad de Loja")
mi_carrera = Carrera("Tranformacion Digital De Empresas", mi_universidad)

# Acceder a los atributos
print(mi_universidad.nombre)
print(mi_carrera.nombre)
print(mi_carrera.universidad.nombre)